css/panel-manager-style.css

```diff
@@ -0,0 +1,50 @@
+/*.panel-manager {
+    display: none;
+    bottom: 30px;
+    right: 0px;
+    top: 30px;
+    height: auto;
+    width: 200px;
+    overflow-y: scroll;
+    position: fixed;
+}*/
+.panel-manager {
+    display: none;
+}
+#slidebar {
+    position: fixed;
+    top: 70px;
+    right: 249px;
+    /*transform: rotate(-90deg);*/
+}
+#slidebar button {
+    border-radius: 4px 0 0 4px;
+    border-right: none;
+    height: 123px;
+    width: 30px;
+    /*border-right: none;*/
+}
+#slider {
```

Comment:
The id "slider" might be a bit too common and risky to use here. Is there a more specfic name for it?